clients/cli/src/config.rs

```diff
@@ -48,9 +48,9 @@ fn signers_of(
     name: &str,
     wallet_manager: &mut Option<Rc<RemoteWalletManager>>,
 ) -> Result<Option<SignersOf>, Box<dyn std::error::Error>> {
-    if let Some(values) = matches.values_of(name) {
+    if let Some(values) = matches.try_get_many(name).ok().flatten() {
```

Comment:
```suggestion
    if let Some(values) = matches.try_get_many::<String>(name).ok().flatten() {
```
In clap-v3, we generally always have to specify the type that we are parsing for. We should parse this to a proper type rather than a `String`, but that will require update the updating the validation function `is_valid_signer`, which will follow after https://github.com/solana-labs/solana-program-library/pull/7447. So let's just update it to `String` here for now.